azure-ipam/logger/logger.go

```diff
@@ -6,56 +6,48 @@ import (
 	"github.com/pkg/errors"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
 type Config struct {
-	Level            string
-	OutputPaths      string // comma separated list of paths
-	ErrorOutputPaths string // comma separated list of paths
+	Level       string // Debug by default
+	Filename   	string
```

Comment:
`FilePath` is probably more connect naming

Reply:
I followed lumberjack naming convention. But have no objection to change it to FilePath